packages/feedback/jest.config.ts

```diff
@@ -0,0 +1,17 @@
+import type { Config } from '@jest/types';
+import { jsWithTs as jsWithTsPreset } from 'ts-jest/presets';
```

Comment:
I see replay also uses the same jest config, but this differs from all of other jest configs. Do you think it's worthwhile to try to align it?